extension/background/languages.js

```diff
@@ -0,0 +1,32 @@
+this.languages = {};
+
+this.languageList = (function() {
```

Comment:
Each file should just export a variable that matches the name of the file. So it should be `this.languages = (function() ...)`

Reply:
Fixed.